src/validation/rules/recommended/recommendedPostfixForUpdateModels.rule.ts

```diff
@@ -55,6 +56,8 @@ export function RecommendedPostfixForUpdateModels(context: ValidationContext): A
       node.fields.forEach((fieldDefinition) => {
         if (
           fieldDefinition.type.kind === ASTNodeKind.NAMED_TYPE &&
+          fieldDefinition.type.name.value &&
+          !specifiedScalarTypes.includes(fieldDefinition.type.name.value) &&
```

Comment:
I expect same migration to set maybe we can move migration to Set in separate or after all prs?